ucp/install-sandbox-2.md

```diff
@@ -0,0 +1,281 @@
+---
+title: Evaluate DDC in a sandbox deployment
+description: Learn how to push Docker images to your private registry and deploy them to your cluster.
+keywords: "Docker Datacenter, registry, orchestration"
+---
+
+# Evaluate DDC in a sandbox deployment
+
+This tutorial assumes that you have
+[installed and configured](install-sandbox.md) a two-node Docker Datacenter
+installation including both UCP and DTR using the instructions
+[here](install-sandbox.md). If you haven't done this, we can't promise that this
+tutorial workflow will work exactly the same.
+
+In the second half of this tutorial, we'll walk you through a typical deployment
+workflow using your sandbox installation of DDC as if it was a production
+instance installed on your organization's network.
+
+
+Over the course of this tutorial, we will:
+
+- Create a repository in DTR
+- Set up certificates or set insecure flag
+- Pull a Docker image, tag it and push it to your DTR repo.
+- Edit the Docker image you just pulled, tag it and push it to your DTR repo.
+- Set up your shell so you can interact with Docker objects in UCP using the command line
+- Use UCP to deploy your edited image to a node
+
+
+## Step 1: Set --insecure registry or set up DTR trust and login
+
+Next, we'll set up a security exception that allows a the Docker-machine hosts
```

Comment:
Since this isn't part of the other doc anymore, probably should change "next" to "first".